services/web/server/src/simcore_service_webserver/director_v2/_comp_run_collections_service.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from aiohttp import web
+from models_library.computations import CollectionRunID
+
+from ..db.plugin import get_database_engine
+from . import _comp_runs_collections_repository
+from ._comp_runs_collections_models import CompRunCollectionDBGet
+
+_logger = logging.getLogger(__name__)
+
+
+async def create_comp_run_collection(
+    app: web.Application,
+    client_or_system_generated_id: str,
+    client_or_system_generated_display_name: str,
+    generated_by_system: bool,
+) -> CollectionRunID:
+    """raises: ProjectNotFoundError"""
+    async with get_database_engine(app).acquire() as conn:
```

Comment:
You should not use aiopg in new code!
As an example, your repository api needs to align with this services/web/server/src/simcore_service_webserver/api_keys/_repository.py